### ISSUE TITLE
Will you share your code on emsemble model

### ISSUE THREAD

**Issue by gaopeng-eugene** [Cadene/vqa.pytorch#15]

**Comment by Cadene** [comment]
Sorry, we don't plan to share our code on ensemble model. We simply calculated the average predictions (after the softmax).

**Comment by gaopeng-eugene** [comment]
Thank you
